main.cpp

```diff
@@ -127,100 +173,113 @@ struct CarWash
     int totalNumMilkOrSubstitutes = 7;
-    //number of sweeteners
     int numSweeteners = 4;
-    //price of coffee per cup
     double coffeeCupPrice = 7.45;
 
-    //make coffee
     void makeCoffee(int getMilkOrSubstitute, int getCoffee, bool checkNeedSweetener = false);
-    //charge customers
     double charge (double getCupPrice = 7.45);
-    //add flavourings to made coffee
     bool addFlavour(bool needsFlavour = true, int getFlavour = 4);
 };
 
-/*
-Thing 2) Theater
-5 properties:
-    1) number of ticket booths (int)
-    2) variety of snacks and drinks (double)
-    3) number of seats (int)
-    4) name of theater (std::string)
-    5) price of ticket per person (float)
-3 things it can do:
-    1) sell tickets
-    2) show movies
-    3) sell snacks and drinks
- */
+void CoffeeCorner::makeCoffee(int getMilkOrSubstitute, int getCoffee, bool checkNeedSweetener)
+{
+    if (checkNeedSweetener == true)
+    {
+        numSweeteners -= 1;
+    }
+
+    getMilkOrSubstitute = totalNumMilkOrSubstitutes;
+    getCoffee = 1;
+    std::cout << "Coffee Ready!\n" << "Coffee Type: " << totalNumMilkOrSubstitutes << " X" << getCoffee << "\n";
+}
+
+double CoffeeCorner::charge(double getCupPrice)
+{
+    int totalNumCups = 2;
+    getCupPrice = coffeeCupPrice;
+    double totalCharge = totalNumCups * getCupPrice;
+    std::cout << "Your charge is: " << totalCharge;
+    return totalCharge;
+}
+
+bool CoffeeCorner::addFlavour(bool needsFlavour, int getFlavour)
+{
+    if (needsFlavour == true)
+    {
+        getFlavour = 2;
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
 
 struct Theater
 {
-    //number of ticket booths
     int numTicketBooths = 3;
-    //variety of snacks and drinks
     double totalNumSnacksAndDrinks = 43;
-    //number of seats
     int totalNumSeats = 156;
-    //name of theater
     std::string theaterName = "Visionary Pictures";
-    //price of ticket per person
-    float ticketPrice = 24.99f;
+    double ticketPrice = 24.99;
 
-    //sell tickets
-    double sellTicket(int numTickets = 3, float ticketPrice = 24.99f);
-    //show movies
+    double sellTicket(int numTickets = 3, double ticketPrice = 24.99);
     void playMovie(int loadMovie, bool lightsOnOrOff, bool trailerOrFeature = true);
-    //sell snacks and drinks
     double sellRefreshments(bool sellSnacks, int snackType, bool sellDrinks = true, int drinkType = 3);
 };
 
-/*
-Thing 3) Gym
-5 properties:
-    1) number of showers (double)
-    2) number of weights (double)
-    3) price of monthly subscription (float)
-    4) names of personal trainers (std::string)
-    5) average number of people per day (float)
-3 things it can do:
-    1) sell monthly subscriptions
-    2) provide personal training sessions
-    3) monitor customer frequency
- */
+double Theater::sellTicket(int numTickets, double ticketPriceCharge)
+{
+    double totalPrice = numTickets * ticketPriceCharge;
+    return totalPrice;
+}
```

Comment:
There is no need for the temporary variable.
`return numTickets * ticketPriceCharge;`
- [ ] revise